libs/ledger-live-common/src/families/cardano/buildSubAccounts.ts

```diff
@@ -102,9 +106,7 @@ const mapTxToTokenAccountOperation = ({
         ),
         blockHeight: tx.blockHeight,
         date: new Date(tx.timestamp),
-        extra: {
-          memo,
-        },
+        extra: extra,
```

Comment:
Cleaner syntax:
`extra: memo ?? {}`

Reply:
doesn't seem equivalent , it's more like `extra: memo ? {memo} : {}` 🤔